platform/core/src/classes/HotkeysManager.js

```diff
@@ -76,6 +77,49 @@ export class HotkeysManager {
     }
   }
 
+  async getHotkeys() {
+    try {
+      const hotKeysResponse = await nlApi.get("/api/hotkeys/$self/");
+      if(hotKeysResponse.status !== 200){
+        throw new Error(
+          'Unable to get hotkeys'
+        );
+      }
+      return hotKeysResponse.data
+    } catch (error) {
+      const { UINotificationService } = this._servicesManager.services;
+      UINotificationService.show({
+        title: 'Hotkeys Manager',
+        message: 'Error while getting hotkeys',
+        type: 'error',
+      });
+    }
+  }
+
+  async saveHotkeys(hotkeyDefinitions = []) {
+    try {
+      // console.log(hotkeyDefinitions);
+      const hotKeysResponse = await nlApi.put("/api/hotkeys/$self/",
+        {hotkeys: {
+          hotkeyDefinitions,
+        }},
+      );
+      if(hotKeysResponse.status !== 200){
```

Comment:
If we have an error from the backend, does it come into catch closure?

Reply:
Yes, it does. I tested by changing the response code here, and the catch block does execute